README.rst

```diff
@@ -75,8 +75,29 @@ Feature extraction:
 
 Installation
 ================
+
+Packages
+---------
 In order to use this, you need to install a few packages first:
 
+You might need to install some other dependencies depending on your operating
+system.  These are for ``scipy`` and ``numpy``.  The fastest way is to install
+the stock packages,
+
+``sudo apt-get install python-numpy python-scipy``
```

Comment:
Perhaps mention that if you do this, you need to pass --system-site-packages when creating your virtualenv?